packages/eslint/src/migrations/update-17-2-0/simplify-eslint-patterns.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ProjectConfiguration,
+  Tree,
+  formatFiles,
+  getProjects,
+  updateProjectConfiguration,
+} from '@nx/devkit';
+
+export default async function update(tree: Tree) {
+  const projects = getProjects(tree);
+
+  for (const [projectName, projectConfiguration] of projects) {
+    let needsUpdate = false;
+
+    for (const [targetName, targetConfig] of Object.entries(
+      projectConfiguration.targets ?? {}
+    )) {
+      // this will be in a broken state after the package is globally renamed
```

Comment:
It's not 100% clear to me what this comment means sorry?

Are you about to globally rename the package?

Reply:
What this means is after this command is run:
```
await replaceNrwlPackageWithNxPackage(tree, '@nx/linter', '@nx/eslint');
```
the executor is now set to `@nx/eslint:eslint` which is an invalid name so we need to change it to `@nx/eslint:lint`

How would you rephrase this?